libs/@guardian/react-crossword/src/components/Clues.tsx

```diff
@@ -1,9 +1,6 @@
 import { css } from '@emotion/react';
-import type {
-	CrosswordEntry,
-	CurrentEntryId,
-	Theme,
-} from '../@types/crossword';
+import type { CAPIEntry } from '../@types/CAPI';
```

Comment:
this feels like a strange change to need to make - maybe we need to check the types